### PR TITLE
Lex Migration - MW & ET Staging

### PR DESCRIPTION
**Pull request by sinehome**
Migrating Malawi and Ethiopia helplines' staging accounts to Lex. 

Malawi
- Adding .hcl files
- Creating bot for youth testing

Ethiopia
- Adding .hcl files

### CODE REVIEW

File: twilio-iac/helplines/mw/common.hcl [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    default_autopilot_chatbot_enabled = false
+    task_language                     = "en"
+    helpline_language                 = "en"
+    voice_ivr_language                = ""
+    contacts_waiting_channels         = ["facebook"]
+    enable_post_survey                = false
+    
+   
+    lex_bot_languages  = {
+      en : ["pre_survey"],
```
Comment (janorivera) [inline twilio-iac/helplines/mw/common.hcl:20]: both bots should be under "en_MW", ideally the "en" slot_types and intents should not be modified unless you want to modify all chatbots using those ones.

File: twilio-iac/helplines/configs/lex/en/intents/pre_survey.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
         "priority": 3,
         "description": "Caller's gender",
         "slot_constraint": "Required",
-        "slot_type": "gender_girl_boy",
+        "slot_type": "gender",
```
Comment (janorivera) [inline twilio-iac/helplines/configs/lex/en/intents/pre_survey.json:57]: I actually think that these are good changes, this and the one made to the gender slot type

File: twilio-iac/helplines/mw/configs/lex/en_MW/intents/pre_survey.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        "slot_type": "yes_no",
+        "value_elicitation_prompt": {
+          "max_attempts": 2,
+          "content": "Do we have your permission to collect personal information? Please answer Yes or No.",
+          "content_type": "PlainText"
+        }
+      },
+      "name": {
+        "priority": 2,
+        "description": "Caller's name",
```
Comment (janorivera) [inline twilio-iac/helplines/mw/configs/lex/en_MW/intents/pre_survey.json:33]: you might be missing the  "slot_constraint": "Required" here

File: twilio-iac/helplines/mw/common.hcl [hunk 4]
```diff
@@ -0,0 +1,69 @@
+locals {
+  defaults_config_hcl = read_terragrunt_config(find_in_parent_folders("defaults.hcl"))
+  defaults_config     = local.defaults_config_hcl.locals
+  config              = merge(local.defaults_config, local.local_config)
+
+ 
+  local_config = {
+    helpline                          = "SaferNet"
```
Comment (janorivera) [inline twilio-iac/helplines/mw/common.hcl:8]: I think SaferNet is BR?

File: twilio-iac/helplines/mw/common.hcl [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    old_dir_prefix                    = "safernet"
+    definition_version                = "mw-v1"
+    default_autopilot_chatbot_enabled = false
+    task_language                     = "en"
+    helpline_language                 = "en"
+    voice_ivr_language                = ""
+    contacts_waiting_channels         = ["facebook"]
```
Comment (janorivera) [inline twilio-iac/helplines/mw/common.hcl:15]: This one can be removed

**Review by janorivera — COMMENTED** [review]
OK I left some comments!

**Comment by janorivera** [comment]
@sinehome  I think you are missing the ET files right?

File: twilio-iac/helplines/mw/common.hcl (hunk 1 above)
Comment (sinehome) [reply twilio-iac/helplines/mw/common.hcl]: I moved this to en_MW and it gave me an error when running the plan

**Comment by sinehome** [comment]
@janorivera I have added ET files and here's the TF plan

[ET_TF_Plan_STG.pdf](https://github.com/techmatters/flex-plugins/files/12369613/ET_TF_Plan_STG.pdf)

File: twilio-iac/helplines/et/common.hcl [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+ 
+  local_config = {
+    helpline                          = "Adama Child Helpline"
+    old_dir_prefix                    = "adama"
+    definition_version                = "et-v1"
+    default_autopilot_chatbot_enabled = false
+    task_language                     = "en"
+    helpline_language                 = "en"
+    voice_ivr_language                = ""
+    contacts_waiting_channels         = ""
```
Comment (janorivera) [inline twilio-iac/helplines/et/common.hcl:15]: you can remove the contacts_waiting_channels variable

File: twilio-iac/helplines/mw/configs/lex/en_MW/bots/pre_survey.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (janorivera) [inline twilio-iac/helplines/mw/configs/lex/en_MW/bots/pre_survey.json:1]: I think this file can be removed, it will use the one under "en"

**Review by janorivera — COMMENTED** [review]
Just a couple of small changes, after doing those feel free to run the applies on Staging and see if everything is OK, you can then do some testing and make sure that they are working as expected. If everything is alright let me know in the PR and I'll approve it!

**Pull request merged**